pylint/testutils/unittest_linter.py

```diff
@@ -30,6 +30,8 @@ def add_message(
         self,
         msg_id: str,
         line: Optional[int] = None,
+        # pylint: disable=fixme
+        # TODO: Make node non optional
```

Comment:
👍

Reply:
Turns out this is actually because node is optional in pylint itself 😅
I guess we can add that to our mental todo list for 3.0

Reply:
Yeap, that's probably some work too :)